src/libopenrave/kinbodygeometry.cpp

```diff
@@ -1045,9 +1100,13 @@ void KinBody::GeometryInfo::SerializeJSON(rapidjson::Value& rGeometryInfo, rapid
         break;
 
     case GT_Cylinder:
-        orjson::SetJsonValueByKey(rGeometryInfo, "topRadius", GetCylinderTopRadius()*fUnitScale, allocator);
-        orjson::SetJsonValueByKey(rGeometryInfo, "height", GetCylinderHeight()*fUnitScale, allocator);
-        orjson::SetJsonValueByKey(rGeometryInfo, "bottomRadius", GetCylinderBottomRadius()*fUnitScale, allocator);
+        if (IsConicalFrustum()) {
```

Comment:
@superfashi in order to simplify implementation of tools using this format, I think we should standardize to output `topRadius` and `bottomRadius` always, even if they are the same value. And `radius` will not be output anymore, we will only support reading it for back compat purposes.

Reply:
1. Need to fix internal test since now we break the (assumed) invariant of Deserialize -> Serialize to the same data.
2. How about collada serializer/deserializer? Do we add top/bottomRadius change there as well?

Reply:
If 2 is not a consideration, I have pushed 1.